documentation/AuthorizationService.md

```diff
@@ -1,14 +1,15 @@
 ## Authorization Service
 
-The authorization service provides API that allows users to add, retrieve, update the data used by the [GPII OAuth2 security server](https://wiki.gpii.net/w/GPII_OAuth_2_Guide). These data include OAuth clients, users, users' authorization decisions and access tokens used by client credentials grant.
+The authorization service provides API that allows users to add, retrieve, update the data used by the [GPII OAuth2 authorization server](https://wiki.gpii.net/w/GPII_OAuth_2_Guide). These data include OAuth clients, users, authorizations used by various OAuth2 grants.
 
-###APIs for [Authorization Code Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Authorization_Code_Grant)
+### APIs for [Authorization Code Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Authorization_Code_Grant)
 
-#### addAuthorization(userId, oauth2ClientId, selectedPreferences)
-* **description**: Add the authorization decision. Check if the user authorization decision already exists based on the user id and OAuth2 client id. If it doesn't exist, generate an access token and add the authorization decision. If it does exist, do nothing. 
+#### addUserAuthorizedAuthorization(userId, solutionId, oauth2ClientId, selectedPreferences)
+* **description**: Add the authorization. Check if the user authorization already exists based on the user id and OAuth2 client id/solution id. If it doesn't exist, generate an access token and add the authorization. If it does exist, do nothing. This function can be used to add 2 type of authorizations: onboarded solution authorizations and web preferences consumer authorizations. When adding onboarded solution authorizations, the 2rd argument `solutionId` must be provided and the 3rd argument `oauth2ClientId` should be empty. Vice versa when adding web preferences consumer authorizations.
 * **parameters:** 
     * userId: Number. A system generated unique number that identifies the user.
```

Comment:
I think I'm seeing an inconsistency here between `AuthorizationService.md` and `AuthServer.md`. In `AuthorizationService.md`, we describe ids as "Numbers". Whereas in `AuthServer.md`, we are saying "Can be a UUID or any unique string". I think these are the same ids? Is that right?